src/crewai/task.py

```diff
@@ -240,7 +240,9 @@ def _execute_core(
         pydantic_output, json_output = self._export_output(result)
 
         task_output = TaskOutput(
+            name=self.name,
```

Comment:
_Suggestion_: Maybe it makes sense to add to the task's constructor or in the Annotations a way to "add" the name for the Task, maybe fetch the functions_name and set it as the Task name.

Reply:
Good call! Added in 9f27e5cb314e08f762a84c4cc86db821e5399f1f :)